Jul-31 09:26:08.823 [main] DEBUG nextflow.cli.Launcher - $> nextflow run main4.nf -resume
Jul-31 09:26:09.785 [main] DEBUG nextflow.cli.CmdRun - N E X T F L O W  ~  version 25.04.6
Jul-31 09:26:09.983 [main] DEBUG nextflow.plugin.PluginsFacade - Setting up plugin manager > mode=prod; embedded=false; plugins-dir=/root/.nextflow/plugins; core-plugins: nf-amazon@2.15.0,nf-azure@1.16.0,nf-cloudcache@0.4.3,nf-codecommit@0.2.3,nf-console@1.2.1,nf-google@1.21.1,nf-k8s@1.0.0,nf-tower@1.11.4,nf-wave@1.12.1
Jul-31 09:26:10.484 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Enabled plugins: []
Jul-31 09:26:10.499 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Disabled plugins: []
Jul-31 09:26:10.527 [main] INFO  org.pf4j.DefaultPluginManager - PF4J version 3.12.0 in 'deployment' mode
Jul-31 09:26:10.653 [main] INFO  org.pf4j.AbstractPluginManager - No plugins
Jul-31 09:26:10.728 [main] DEBUG nextflow.config.ConfigBuilder - Found config home: /root/.nextflow/config
Jul-31 09:26:10.742 [main] DEBUG nextflow.config.ConfigBuilder - Found config local: /mnt/e/rare-disease-variant-annotation/nextflow.config
Jul-31 09:26:10.760 [main] DEBUG nextflow.config.ConfigBuilder - Parsing config file: /root/.nextflow/config
Jul-31 09:26:10.766 [main] DEBUG nextflow.config.ConfigBuilder - Parsing config file: /mnt/e/rare-disease-variant-annotation/nextflow.config
Jul-31 09:26:11.728 [main] DEBUG n.secret.LocalSecretsProvider - Secrets store: /root/.nextflow/secrets/store.json
Jul-31 09:26:11.737 [main] DEBUG nextflow.secret.SecretsLoader - Discovered secrets providers: [nextflow.secret.LocalSecretsProvider@5c089b2f] - activable => nextflow.secret.LocalSecretsProvider@5c089b2f
Jul-31 09:26:11.757 [main] DEBUG nextflow.config.ConfigBuilder - Applying config profile: `standard`
Jul-31 09:26:14.076 [main] DEBUG nextflow.config.ConfigBuilder - Applying config profile: `standard`
Jul-31 09:26:14.481 [main] DEBUG nextflow.cli.CmdRun - Applied DSL=2 from script declaration
Jul-31 09:26:14.536 [main] DEBUG nextflow.cli.CmdRun - Launching `main4.nf` [condescending_kalman] DSL2 - revision: fee50f16a5
Jul-31 09:26:14.543 [main] DEBUG nextflow.plugin.PluginsFacade - Plugins default=[]
Jul-31 09:26:14.544 [main] DEBUG nextflow.plugin.PluginsFacade - Plugins resolved requirement=[]
Jul-31 09:26:14.874 [main] DEBUG nextflow.Session - Session UUID: 299dd5e4-533e-4e02-9c3e-c4c5ae27d28a
Jul-31 09:26:14.879 [main] DEBUG nextflow.Session - Run name: condescending_kalman
Jul-31 09:26:14.884 [main] DEBUG nextflow.Session - Executor pool size: 8
Jul-31 09:26:15.017 [main] DEBUG nextflow.file.FilePorter - File porter settings maxRetries=3; maxTransfers=50; pollTimeout=null
Jul-31 09:26:15.106 [main] DEBUG nextflow.util.ThreadPoolBuilder - Creating thread pool 'FileTransfer' minSize=10; maxSize=24; workQueue=LinkedBlockingQueue[-1]; allowCoreThreadTimeout=false
Jul-31 09:26:15.273 [main] DEBUG nextflow.cli.CmdRun - 
  Version: 25.04.6 build 5954
  Created: 01-07-2025 11:27 UTC (12:27 BST)
  System: Linux 5.15.167.4-microsoft-standard-WSL2
  Runtime: Groovy 4.0.26 on OpenJDK 64-Bit Server VM 17.0.10+7
  Encoding: UTF-8 (UTF-8)
  Process: 31454@abishek [127.0.1.1]
  CPUs: 8 - Mem: 3.7 GB (200.4 MB) - Swap: 1 GB (578.8 MB)
Jul-31 09:26:15.565 [main] DEBUG nextflow.Session - Work-dir: /mnt/e/rare-disease-variant-annotation/work [v9fs]
Jul-31 09:26:15.570 [main] DEBUG nextflow.Session - Script base path does not exist or is not a directory: /mnt/e/rare-disease-variant-annotation/bin
Jul-31 09:26:15.654 [main] DEBUG nextflow.executor.ExecutorFactory - Extension executors providers=[]
Jul-31 09:26:15.728 [main] DEBUG nextflow.Session - Observer factory: DefaultObserverFactory
Jul-31 09:26:15.777 [main] DEBUG nextflow.Session - Observer factory (v2): LinObserverFactory
Jul-31 09:26:15.990 [main] DEBUG nextflow.cache.CacheFactory - Using Nextflow cache factory: nextflow.cache.DefaultCacheFactory
Jul-31 09:26:16.230 [main] DEBUG nextflow.util.CustomThreadPool - Creating default thread pool > poolSize: 9; maxThreads: 1000
Jul-31 09:26:17.184 [main] DEBUG nextflow.Session - Session start
Jul-31 09:26:18.191 [main] DEBUG nextflow.script.ScriptRunner - > Launching execution
Jul-31 09:26:19.548 [main] DEBUG nextflow.script.ProcessConfig - Config settings `withName:VEP_ANNOTATION` matches process VEP_ANNOTATION
Jul-31 09:26:19.604 [main] DEBUG nextflow.executor.ExecutorFactory - << taskConfig executor: local
Jul-31 09:26:19.606 [main] DEBUG nextflow.executor.ExecutorFactory - >> processorType: 'local'
Jul-31 09:26:19.628 [main] DEBUG nextflow.executor.Executor - [warm up] executor > local
Jul-31 09:26:19.653 [main] DEBUG n.processor.LocalPollingMonitor - Creating local task monitor for executor 'local' > cpus=8; memory=3.7 GB; capacity=8; pollInterval=100ms; dumpInterval=5m
Jul-31 09:26:19.662 [main] DEBUG n.processor.TaskPollingMonitor - >>> barrier register (monitor: local)
Jul-31 09:26:19.764 [main] DEBUG nextflow.processor.TaskProcessor - Creating process 'VEP_ANNOTATION': maxForks=0; fair=false; array=0
Jul-31 09:26:19.919 [main] DEBUG nextflow.Session - Workflow process names [dsl2]: VEP_ANNOTATION
Jul-31 09:26:19.926 [main] DEBUG nextflow.Session - Igniting dataflow network (2)
Jul-31 09:26:19.953 [main] DEBUG nextflow.processor.TaskProcessor - Starting process > VEP_ANNOTATION
Jul-31 09:26:19.959 [main] DEBUG nextflow.script.ScriptRunner - Parsed script files:
  Script_02b95656ce6ab928: /mnt/e/rare-disease-variant-annotation/main4.nf
  Script_a4b88fdaf9dcfad4: /mnt/e/rare-disease-variant-annotation/workflow/modules/annotation/VEP.nf
Jul-31 09:26:19.960 [main] DEBUG nextflow.script.ScriptRunner - > Awaiting termination 
Jul-31 09:26:19.964 [main] DEBUG nextflow.Session - Session await
Jul-31 09:26:20.626 [Task submitter] DEBUG n.executor.local.LocalTaskHandler - Launch cmd line: /bin/bash -ue .command.run
Jul-31 09:26:20.634 [Task submitter] INFO  nextflow.Session - [72/a1dc83] Submitted process > VEP_ANNOTATION (1)
Jul-31 09:28:52.888 [Task monitor] DEBUG n.processor.TaskPollingMonitor - Task completed > TaskHandler[id: 1; name: VEP_ANNOTATION (1); status: COMPLETED; exit: 127; error: -; workDir: /mnt/e/rare-disease-variant-annotation/work/72/a1dc83096407f2c1255fb01733f850]
Jul-31 09:28:52.976 [Task monitor] DEBUG nextflow.util.ThreadPoolBuilder - Creating thread pool 'TaskFinalizer' minSize=10; maxSize=24; workQueue=LinkedBlockingQueue[-1]; allowCoreThreadTimeout=false
Jul-31 09:28:53.172 [TaskFinalizer-1] DEBUG nextflow.processor.TaskProcessor - Handling unexpected condition for
  task: name=VEP_ANNOTATION (1); work-dir=/mnt/e/rare-disease-variant-annotation/work/72/a1dc83096407f2c1255fb01733f850
  error [nextflow.exception.ProcessFailedException]: Process `VEP_ANNOTATION (1)` terminated with an error exit status (127)
Jul-31 09:28:53.443 [TaskFinalizer-1] ERROR nextflow.processor.TaskProcessor - Error executing process > 'VEP_ANNOTATION (1)'

Caused by:
  Process `VEP_ANNOTATION (1)` terminated with an error exit status (127)


Command executed:

  vep -i SRR32809192.filtered.vcf         --assembly GRCh38         --vcf         --database
      -o SRR32809192.vep.vcf

Command exit status:
  127

Command output:
  (empty)

Command error:
  .command.sh: line 3: -o: command not found

Work dir:
  /mnt/e/rare-disease-variant-annotation/work/72/a1dc83096407f2c1255fb01733f850

Container:
  ensemblorg/ensembl-vep

Tip: you can try to figure out what's wrong by changing to the process work dir and showing the script file named `.command.sh`
Jul-31 09:28:53.523 [TaskFinalizer-1] DEBUG nextflow.Session - Session aborted -- Cause: Process `VEP_ANNOTATION (1)` terminated with an error exit status (127)
Jul-31 09:28:53.545 [main] DEBUG nextflow.Session - Session await > all processes finished
Jul-31 09:28:53.613 [Task monitor] DEBUG n.processor.TaskPollingMonitor - <<< barrier arrives (monitor: local) - terminating tasks monitor poll loop
Jul-31 09:28:53.615 [main] DEBUG nextflow.Session - Session await > all barriers passed
Jul-31 09:28:53.710 [main] DEBUG n.trace.WorkflowStatsObserver - Workflow completed > WorkflowStats[succeededCount=0; failedCount=1; ignoredCount=0; cachedCount=0; pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0; abortedCount=0; succeedDuration=0ms; failedDuration=2m 32s; cachedDuration=0ms;loadCpus=0; loadMemory=0; peakRunning=1; peakCpus=1; peakMemory=0; ]
Jul-31 09:28:53.734 [TaskFinalizer-1] WARN  nextflow.util.SimpleAgent - Got an interrupted exception while taking agent result | java.lang.InterruptedException
Jul-31 09:28:54.045 [main] DEBUG nextflow.cache.CacheDB - Closing CacheDB done
Jul-31 09:28:54.203 [main] DEBUG nextflow.script.ScriptRunner - > Execution complete -- Goodbye
